build.gradle

```diff
@@ -13,7 +13,7 @@ plugins {
 
 group 'jp.co.soramitsu'
 def pluginId = "${group}.sora-plugin"
-version System.getenv('TAG_NAME') ?: "0.5.3"
+version System.getenv('TAG_NAME') ?: "0.5.4-rc3"
```

Comment:
we shouldn't create new tags without real releases, please consider removing all unnecessary tags and replace that with just `0.5.4`, we can make a release right after merge

Reply:
Done.